modules/unitbar.lua

```diff
@@ -63,6 +63,10 @@ function GladiusEx:NewUnitBarModule(name, defaults_arena, defaults_party)
 			self.frame[unit].statusbar:SetStatusBarColor(color.r, color.g, color.b, color.a or 1)
 			self.frame[unit].icon:SetTexture([[Interface\Glues\CharacterCreate\UI-CharacterCreate-Classes]])
 
+			-- V: wtf is "CLASS_BUTTONS"?
+			if not CLASS_BUTTONS then

```

Comment:
Oh, okay. I thing this is just related to backgrounds being removed from such places (same for GetSpecializationInfoByID).
I can confirm the class icon correctly shows.